lib/plugins/aws/lib/naming.test.js

```diff
@@ -423,6 +436,14 @@ describe('#naming()', () => {
       });
   });
 
+  describe('#getLambdaCloudWatchPermissionLogicalId()', () => {
```

Comment:
Maybe we should rename this function to `getLambdaEventsRulePermissionLogicalId` so that it follows the same naming (based on the CloudFormation resource which is used) like e.g. `getLambdaApiGatewayPermissionLogicalId`.